opentreemap/treemap/templates/treemap/settings.js

```diff
@@ -45,10 +45,23 @@ otm.settings.trans = Object.freeze({
     noStreetViewText: '{% trans "Could not load street view for this location" %}',
     treeDetails: '{% trans "Tree Details" %}',
     resourceDetails: '{{ term.Resource.singular }}{% trans " Details" %}',
-    {# this has to be broken into two sections because window.onbeforeunload has a default but confirm() does not #}
+    //{# this has to be broken into two sections because window.onbeforeunload has a default but confirm() does not #}
```

Comment:
That was to fix problems with syntax highlighting in my editor -- its JS mode got confused by the `{#`. Seems harmless to comment the comment.